canonical-web-and-design/vanilla-framework
# Restructure docs sidebar navigation

## Done

- Restructured navigation items to match what is seen in code/examples page
  - Base elements
  - Components
  - Utilities
  - Settings
  - **Resources (NEW)**
- Added tables back into docuemntation and renamed back to **Tables**
- **Resources (NEW)** - Added link to examples page and design UI kit download 👍 

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Scroll down the sidebar navigation in all its glory
- Check items under **Resources** to make sure _Component examples_ links to the correct page and the _Design UI Kit_ is downloadable
- #winning

## Details

- Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/2390

## Screenshots

<img width="721" alt="Screenshot 2019-06-26 at 11 22 08" src="https://user-images.githubusercontent.com/17748020/60172901-8f6c2b00-9805-11e9-8d69-869ec4c285e4.png">
